azure-pipelines.yml

```diff
@@ -0,0 +1,51 @@
+trigger:
+  branches:
+    include:
+    - refs/tags/v*
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: '14.x'
+      checkLatest: true
+
+  - script: |
+        npm install yo
+        npm install 
+    displayName: 'npm install'
+
+  - script: |
+        npm run lint 
+    displayName: 'npm lint'
+    
+  - task: Bash@3
```

Comment:
>task: Bash@3 [](http://example.com/codeflow?start=4&length=12)

please also add the npm publish task, when we get the credentials we can test it out

Reply:
Added.